### PR TITLE
Lots of CDDL fixes

### PR DESCRIPTION
**Pull request by laurencelundblade**

### CODE REVIEW

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -794,91 +819,82 @@ secure element.
...
+oemid = "oemid"
+security-level = "security-level"
+boot-state = "boot-state"
+location = "location"
+age = "age"
+uptime = "uptime"
+nested-eat = "nested-eat"
+submods = "submods"
+
+latitude = "lat""
```
Comment (henkbirkholz) [inline draft-ietf-rats-eat.md]: latitude = "lat"

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -551,18 +559,20 @@ should pick one and prefer that for all their devices.
...
 
-### CDDL
-
-    oemid_claim = (
-    oemid: bstr )
+### oemid CDDL
 
+~~~~CDDL
+oemid-claim = (
+    oemid: bstr
```
[the remark this answers is not in the document]
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md]: This is an OUI or CD, not EUI.

It needs to be possible to use OUIs and such that were obtained 20-30 years ago, so it is OK as is.

File: draft-ietf-rats-eat.md (hunk 1 above)
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md]: Fixed

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -527,10 +532,13 @@ and other are in separate fields.
...
 in CWT in that it describes the authority that created the token.
 
-### CDDL
+### origination CDDL
 
-    origination_claim = (
-    origination: string_or_uri )
+~~~~CDDL
+origination-claim = (
+    origination: string-or-uri
```
[the remark this answers is not in the document]
Comment (laurencelundblade) [reply draft-ietf-rats-eat.md]: It is defined later in the doc roughly as you suggest.

**Pull request merged**